### PR TITLE
docs: add data provider consent spec for protocol devs

### PR DESCRIPTION
**Pull request by H4NLee**
Based on #567, added specs of data provider consent for protocol devs

### CODE REVIEW

File: .gitbook/3-protocol-devs/1-dep-specs/3-data-provider-consent.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    - Myongsik Gong <myongsik_gong@medibloc.org>
+    - Inchul Song <icsong@medibloc.org>
+
+
+## Synopsis
+
+This document is about providing data through [DEP](../../1-users/3-data-exchange/0-about-dep.md).
+Before providers provide their data to consumers, the data must be validated from oracle whether the data is the correct data specified in the deal.
+Providers can send a request for validation to one of the oracles registered in panacea, and the oracle will validate the data using confidential computing without any exposure of the data.
+For the data that has been validated, oracle will issue a certificate to the provider.
```
Comment (youngjoon-lee) [inline .gitbook/3-protocol-devs/1-dep-specs/3-data-provider-consent.md:19]:
```suggestion
As a result of a data validation, oracle will issue a certificate to the provider.
```

**Review by gyuguen — APPROVED** [review]
LGTM

I have a question.

Do we call blockchain names a `Panacea` in docs?
I ask for unification to other documents as well.

**Review by audtlr24 — APPROVED** [review]
Good

**Review by youngjoon-lee — APPROVED** [review]
lgtm

**Comment by gyuguen** [comment]
> > Do we call blockchain names a Panacea in docs?
> > I ask for unification to other documents as well.
> 
> Yes. Let's use Panacea until we have a clear decision for naming. There are still so many discussions/feedbacks about naming, but it's not simple to decide within only the blockchain team.

OK. Thanks

File: .gitbook/3-protocol-devs/1-dep-specs/3-data-provider-consent.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  Certificate certificate = 1;
+}
+```
+
+When this consent is submitted, blockchain will check:
+- if the data is provided by the owner of the data
+- if the data is validated by a registered and active oracle
+- if the data is provided in duplicate
+
+If all checks pass, rewards are distributed to the provider and oracle(more about [incentive](./6-incentives.md)).
```
Comment (inchori) [inline .gitbook/3-protocol-devs/1-dep-specs/3-data-provider-consent.md:72]:
```suggestion
If all checks are passed, rewards are distributed to the provider and oracle(more about [incentive](./6-incentives.md)).
```

**Review by inchori — APPROVED** [review]
LGTM

File: .gitbook/3-protocol-devs/1-dep-specs/3-data-provider-consent.md (hunk 2 above)
Comment (H4NLee) [reply .gitbook/3-protocol-devs/1-dep-specs/3-data-provider-consent.md:72]: I used the `pass` as an intransitive verb which have no direct object.

Comment (inchori) [reply .gitbook/3-protocol-devs/1-dep-specs/3-data-provider-consent.md:72]: Ah, I'm not sure my comment is right. My purpose is change to past verb.

**Pull request merged**